.travis.yml

```diff
@@ -0,0 +1,14 @@
+notifications:
+  email: false
+
+dist: trusty
+sudo: false
+language: python
+python: 2.7
```

Comment:
you can use `tox-travis` and parrallelise your builds https://github.com/MichaelAquilina/S4/blob/master/.travis.yml